### ISSUE TITLE
The clothes don't show up properly.

### ISSUE THREAD

**Issue by lilabyte** [indilo53/gtautil#15]
Hello, I created a pack of clothes for FiveM using gtautil, but if someone puts on clothes with a number greater than 127, nobody except him can see the clothes correctly. Instead you can see various other models of GTA V clothes.

I tried to change reserved slots in this order: 200, 400, 1000, 0. But it didn't work either.

# Screenshots
### Clothe with number 128
![image](https://user-images.githubusercontent.com/38222444/88376435-afba4080-cd9d-11ea-887c-231ba87881bc.png)

I'm invisible because of character editor script
![image](https://user-images.githubusercontent.com/38222444/88376553-f0b25500-cd9d-11ea-8028-77c8fae5e8ad.png)

***

### Clothe with number 127
![image](https://user-images.githubusercontent.com/38222444/88376689-396a0e00-cd9e-11ea-8204-5dca99fe4572.png)

![image](https://user-images.githubusercontent.com/38222444/88376747-556daf80-cd9e-11ea-996d-6f2cb0218021.png)

**Comment by YisusOnDev** [comment]
Same problem, seems like gta gets stuck on +111

**Comment by NotWasooked** [comment]
This is an issue we had. We just resorted to having the clothes resource get pruned

**Comment by YisusOnDev** [comment]
So u fixed that problem?, because i has the same problem on my server...

**Comment by NotWasooked** [comment]
> So u fixed that problem?, because i has the same problem on my server...

No, Like it says above we just resorted to pruning the amount of clothing that we had.

**Comment by corrupt3k** [comment]
I also have this issue. Anything after 127 is not showing correctly for other players. Any ideas?

**Comment by Honox17** [comment]
hello, anyone have fixed that problem ?

**Comment by Disquse** [comment]
The game's code don't expect more than 127 clothing items in a single `#mt` file. This *could* be adjusted up to 255 I believe, but we can't be sure about side effects. For now the best solution would be to split your pack into multiple `#mt` files.

**Comment by Naben-dev** [comment]
> The game's code don't expect more than 127 clothing items in a single `#mt` file. This _could_ be adjusted up to 255 I believe, but we can't be sure about side effects. For now the best solution would be to split your pack into multiple `#mt` files.

What do you think about #mt files i dont know what is it

**Comment by HeyMari** [comment]
> The game's code don't expect more than 127 clothing items in a single `#mt` file. This _could_ be adjusted up to 255 I believe, but we can't be sure about side effects. For now the best solution would be to split your pack into multiple `#mt` files.

I would also like to know how..

**Comment by Ryzeeeev** [comment]
Any help for this because i have this problem.

**Comment by Disquse** [comment]
> Any help for this because i have this problem.

There can't be any help, that's how the game works. You can't alter this behavior.

> What do you think about #mt files i dont know what is it

If you don't know what's `ymt`s (or `#mt` files, if we name it properly), then you should learn basics of modern RAGE modding, how clothing metadata works.

**Comment by Wilsoonx** [comment]
> The game's code don't expect more than 127 clothing items in a single `#mt` file. This _could_ be adjusted up to 255 I believe, but we can't be sure about side effects. For now the best solution would be to split your pack into multiple `#mt` files.

hi mate, do you know how to adjust the items upto 255?

**Comment by Disquse** [comment]
> hi mate, do you know how to adjust the items upto 255?

It is not something you can do yourself most likely. Unless you know how to reverse engineer GTA5 and have some time.
